### ISSUE TITLE
typo in decaying the learning rate example

### ISSUE THREAD

**Issue by mikowals** [tensorflow/tensorflow#141]
From example in [docs](http://tensorflow.org/api_docs/python/train.md#decaying_the_learning_rate):

> learning_rate = tf.exponential_decay(starter_learning_rate, global_step,
                                     100000, 0.96, staircase=True)

should match function definiton above `tf.train.exponential_decay`.

**Comment by vrv** [comment]
Thanks, fixed this in our internal repo -- will push out the fix some time this week.
